maintainers/maintainer-list.nix

```diff
@@ -17079,6 +17079,14 @@
     githubId = 1922770;
     name = "Tim Engler";
   };
+  redhawk = {
+    email = "redhawk76767676@gmail.com";
+    github = "Redhawk18";
+    githubId = 77415970;
+    name = "Redhawk";
+  };
+
+
```

Comment:
Remove the extra blank lines for consistency in the entire file.

Reply:
I'm having trouble amending the commit.

Reply:
```bash
─redhawk💾Mythra─ ~/code/nixpkgs 
 └$ git status
On branch master
Your branch and 'origin/master' have diverged,
and have 1 and 1 different commits each, respectively.
  (use "git pull" if you want to integrate the remote branch with yours)

nothing to commit, working tree clean

─redhawk💾Mythra─ ~/code/nixpkgs 
 └$ git pull

```
Git pull keeps hanging

Reply:
Oh I let it do whatever git was doing and now its pulling, I hope it's working

Reply:
Nope I created a git nightmare